db/dcrpg/internal/common.go

```diff
@@ -32,41 +27,3 @@ const (
 		timestamptz TIMESTAMPTZ
 	);`
 )
-
-// makeARRAYOfTEXT parses the string slice into a PostgreSQL-syntax ARRAY
-// string.
-func makeARRAYOfTEXT(text []string) string {
-	if len(text) == 0 {
-		return "ARRAY['']"
-	}
-	buffer := bytes.NewBufferString("ARRAY[")
-	for i, txt := range text {
-		if i == len(text)-1 {
-			buffer.WriteString(`'` + txt + `'`)
-			break
-		}
-		buffer.WriteString(`'` + txt + `', `)
-	}
-	buffer.WriteString("]")
-
-	return buffer.String()
-}
-
-func makeARRAYOfBIGINTs(ints []uint64) string {
-	if len(ints) == 0 {
-		return "'{}'::BIGINT[]" // cockroachdb: "ARRAY[]:::BIGINT[]"
```

Comment:
Oh, I wonder if I was doing this string gen stuff because cockroachdb sucks with the lib/pq driver.  Ah well, maybe cockroach will be more compatible someday.  The right way to do this is how you got it now, with pq.Array/etc.